vividus-allure-adaptor/src/main/java/org/vividus/report/allure/model/AllureCategory.java

```diff
@@ -31,6 +33,14 @@ public AllureCategory(String name, List<Status> matchedStatuses)
         this.matchedStatuses = matchedStatuses;
     }
 
+    @ConstructorProperties({ "name", "statuses" })
```

Comment:
is it really required?

Reply:
> is it really required?

As far as I know, if we don't have a default constructor - yes.
I also don't think that Jackson correctly select the right constructor even if we have the default one

Reply:
feel free to refactor to use setters instead of constructors

Reply:
> feel free to refactor to use setters instead of constructors

Done. Pay attention, especially for Allure mechanisms I have been forced to preserve the old getter for `matchedStatuses` variable name.